docs/source/getting-started/typical-workflow.rst

```diff
@@ -236,27 +248,37 @@ Here's a differentially private mean:
         .. literalinclude:: code/typical-workflow-context.rst
             :language: python
             :dedent:
-            :start-after: mean
-            :end-before: /mean
+            :start-after: sum
+            :end-before: /sum
 
     .. tab-item:: Framework API (Python)
         :sync: framework
 
         .. literalinclude:: code/typical-workflow-framework.rst
             :language: python
             :dedent:
-            :start-after: mean
-            :end-before: /mean
+            :start-after: sum
+            :end-before: /sum
 
     .. tab-item:: Framework API (R)
         :sync: r
 
         .. literalinclude:: code/typical-workflow-framework.R
             :language: r
-            :start-after: mean
-            :end-before: /mean
+            :start-after: sum
+            :end-before: /sum
+
+Altogether, the data flow of this analysis looks like:
+
+.. Diagram source: https://docs.google.com/drawings/d/1W4l9x3UM3hbVLWlC0nzijqgaQ31wY5ERebp8jkYy1yc/edit
+
+.. image:: code/typical-workflow-diagram.svg
+    :width: 100%
+    :alt: Diagram representing typical data flow with OpenDP, raw data to differentially private releases. 
```

Comment:
Not an absolute, but I liked a diagram up front, as an outline of what is to follow. You may be giving it a different role?